Resources/Prototypes/Entities/Objects/Weapons/Guns/SMGs/smgs.yml

```
@@ -347,28 +349,28 @@
-        whitelist:
-          tags:
-            - MagazineMagnumSubMachineGun
-      gun_chamber:
-        name: Chamber
-        startingItem: CartridgeMagnumRubber
-        priority: 1
-        whitelist:
-          tags:
-            - CartridgeMagnum
\ No newline at end of file
+#- type: entity
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)